src/backend/optimizer/plan/planner.c

```diff
@@ -8766,3 +8733,9 @@ void compute_jit_flags(PlannedStmt* pstmt)
 			pstmt->jitFlags |= PGJIT_DEFORM;
 	}
 }
+
+static void standard_planner_compute_jit_flags(PlannedStmt* pstmt)
```

Comment:
Why do we need this function? It is called from one place only and contains only one function call. What about replacing the `standard_planner_compute_jit_flags` call with the function body?

Reply:
This function hides the JIT related guts from the main planner code.

I think this call inside the `standard_planner()`:
```
standard_planner_compute_jit_flags(pstmt);
```
is more preferable then 
```
compute_jit_flags(pstmt, jit_above_cost, jit_inline_above_cost, jit_optimize_above_cost);
```
It doesn't pollute planner code with JIT entities and it makes `standard_planner()` more readable.

Plus, if something changes in the JIT part, more likely we'll not need to touch the `standard_planner()` code in this case.